systemtest/src/main/java/io/strimzi/systemtest/MessagingBaseST.java

```diff
@@ -176,9 +176,9 @@ private boolean allowParameter(String minimalVersion) {
         Matcher current = pattern.matcher(Environment.ST_KAFKA_VERSION);
         Matcher minimal = pattern.matcher(minimalVersion);
         if (current.find() && minimal.find()) {
-            return Integer.valueOf(current.group("major")) >= Integer.valueOf(minimal.group("major"))
-                    && Integer.valueOf(current.group("minor")) >= Integer.valueOf(minimal.group("minor"))
-                    && Integer.valueOf(current.group("micro")) >= Integer.valueOf(minimal.group("micro"));
+            return Integer.parseInt(current.group("major")) >= Integer.parseInt(minimal.group("major"))
```

Comment:
why are we replacing `valueOf` to `parseInt` ? Basically what i know these static methods are the same only one difference. `ValueOf -> returns Integer` and `ParseInt returns int`.

Reply:
I think you need to use `compareTo()` when using `Integer` as it is not a primitive data type. For `int` you can use `>=` operator. What I think, the compiler unboxes the `Integer` value into `int` whenever it sees you want to use binary compare operators. 
The `valueOf()` internally calls `parseInt()` but it returns `Integer` (`parseInt()` returns `int`), what results in you have `int` (from `parseInt()`) value boxed into `Integer` (from `valueOf()`) and immediately unboxed back to `int` (by `>=` operator).